src/renderer/pages/game-details/delete-modal.tsx

```diff
@@ -0,0 +1,55 @@
+import { Button, Modal } from "@renderer/components";
+import { useTranslation } from "react-i18next";
+import * as styles from "./delete-modal.css";
+
+type DeleteModalProps = {
+  visible: boolean;
+  title: string;
```

Comment:
Why do you have to pass `title` and `description` as props if they're always the same and this component is going to be used in a single place?